### DISCUSSION TITLE
Which parameter should I use for a more precise cut?

### DISCUSSION THREAD

**Discussion by yusufenser** [WyattBlue/auto-editor#438] [1. Help] [answered]
It performs automatic cutting. really amazing. I thank those who contributed. But I want to make a more precise cut. When I look after the automatic video cut, there are quiet areas, albeit small. I don't know what parameter I should use to prevent them from happening.

**Comment by WyattBlue** [comment, marked as answer]
@yusufenser I would recommend setting the margin to 0. Setting "mincut" to a value around half a second would also help remove small quiet second.

```
auto-editor --margin 0 --edit audio:mincut=0.5sec
```

**Reply by yusufenser** [reply]
yes it worked. You are great. Thank you
